pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -22,6 +22,10 @@ const (
 	// AnnotationBuildRefSecret is an annotation that tells the Build Controller to reconcile on
 	// events of the secret only if is referenced by a Build in the same namespace
 	AnnotationBuildRefSecret = "build.build.dev/referenced.secret"
+
+	// AnnotationBuildRefRepository is a label key that tells the Build Controller to check whether remote repository exists or not.
+	// If its value is true, controller checks remote repository, otherwise, it won't be checked
+	AnnotationBuildRefRepository = "build.build.dev/verify.repository"
```

Comment:
This constant name doesn't really signify what it is being used for, how about something like "AnnotationBuildVerifyRepository" ?

Reply:
Done!